backend/src/test/java/com/woowacourse/kkogkkog/member/domain/repository/MemberRepositoryTest.java

```diff
@@ -0,0 +1,42 @@
+package com.woowacourse.kkogkkog.member.domain.repository;
+
+import static com.woowacourse.kkogkkog.support.fixture.domain.MemberFixture.LEO;
+import static com.woowacourse.kkogkkog.support.fixture.domain.MemberFixture.ROOKIE;
+import static com.woowacourse.kkogkkog.support.fixture.domain.WorkspaceFixture.KKOGKKOG;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.woowacourse.kkogkkog.member.domain.Member;
+import com.woowacourse.kkogkkog.member.domain.Workspace;
+import com.woowacourse.kkogkkog.support.repository.RepositoryTest;
+import java.util.List;
+import javax.persistence.EntityManager;
+import javax.persistence.PersistenceContext;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+
+@RepositoryTest
+class MemberRepositoryTest {
+
+    @PersistenceContext
+    private EntityManager entityManager;
+
+    @Autowired
+    private WorkspaceRepository workspaceRepository;
+    @Autowired
+    private MemberRepository memberRepository;
+
+    @Test
+    @DisplayName("회원의 nickname이 앞글자부터 동일한 nickname을 찾는다.")
+    void findByNickname() {
+        Workspace workspace = workspaceRepository.save(KKOGKKOG.getWorkspace());
+        memberRepository.save(ROOKIE.getMember(workspace));
+        memberRepository.save(LEO.getMember(workspace));
+
+        entityManager.clear();
+
+        List<Member> members = memberRepository.findByNickname("루");
```

Comment:
1. 저장되는 회원 두 명 중 닉네임이 "루"로 시작하는 경우가 1명이라는 점을 알려면 Fixture를 타고 들어가야 한다는 점이 좀 아쉽네요. 

2. 저 사이의 여백을 제거하는건 어떤가요.

```suggestion
        Workspace workspace = workspaceRepository.save(KKOGKKOG.getWorkspace());
        memberRepository.save(ROOKIE.getMember(workspace));
        memberRepository.save(LEO.getMember(workspace));
        entityManager.clear();

        List<Member> members = memberRepository.findByNickname("루");
```

Reply:
1번에 대한 내용은 Fixture 생성시 매개변수에 닉네임을 넣어주는 방법을 사용하면 어느정도 개선이 될거라 생각해서 Fixture 개선 PR에서 같이 고려해볼게요 ㅎㅎ

2번에 대한 코멘트는 반영했습니다~